website/content/docs/connect/gateways/api-gateway/configuration/gatewaypolicy.mdx

```diff
@@ -0,0 +1,257 @@
+---
+layout: docs
+page_title: GatewayPolicy configuration reference
+description: Learn about the configuration options for the GatewayPolicy configuration resource. GatewayPolicy resources define API gateway polcies for Consul service mesh on Kubernetes.
+---
+
+# GatewayPolicy
+
+This topic provides reference information for the API gateway policy resource, which defines API gateway policies for Consul service mesh on Kubernetes-orchestrated networks.
```

Comment:
should we also include something about how the policy can _only_ target listeners on a gateway and not the entire gateway itself?